### PR TITLE
Cleanup Postgresql instructions:

### PR DESCRIPTION
**Pull request by shivharis**
1. Add rule to install postgressql server for deb.rst
2. Fix minor grammar in rhel6.rst and rhel7.rst

### CODE REVIEW

**Comment by shivharis** [comment]
there is no need to install the -contrib or the -devel packages for debian (this is unlike rhel, maybe rhel also does not need it, but will require testing to verify - not done)

**Comment by shivharis** [comment]
Script used to test:#! /bin/bash

curl -s https://packagecloud.io/install/repositories/StackStorm/stable/script.deb.sh | sudo bash
sudo apt-get install -y st2 st2mistral

# Create Mistral DB in PostgreSQL
cat << EHD | sudo -u postgres psql
CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
CREATE DATABASE mistral OWNER mistral;
EHD

# Setup Mistral DB tables, etc.
/opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
# Register mistral actions
/opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate

**Comment by shivharis** [comment]
Ran without postgresql, then ran after installing postgresql. Results: Fail and Pass respectively.

**Comment by shivharis** [comment]
The # symbol in the comments end up making the font bigger. Darn.

**Comment by lakshmi-kannan** [comment]
Good catch. Luckily, the script installs postgres https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh#L256

Your best bet is to just use ```curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password=<CHANGEME>
```

The script goes through our CI.

File: docs/source/install/deb.rst [hunk 1]
```diff
@@ -59,6 +59,7 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 
     sudo apt-get install -y mongodb-org
     sudo apt-get install -y rabbitmq-server
+    sudo apt-get install -y postgresql
```
Comment (armab) [inline docs/source/install/deb.rst]: :+1: and Ouch

Luckily, mistake appeared not so long ago, with MongoDB 3.2 changes:
[#213 Update documentation for MongoDB 3.2 changes, reduce duplication / copy & paste](https://github.com/StackStorm/st2docs/pull/213)

Comment (Kami) [inline docs/source/install/deb.rst]: ouch, sorry and good catch :+1:

**Pull request merged**